src/sendmail/mail_sender.py

```diff
@@ -0,0 +1,58 @@
+import os
+import mailtrap as mt
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+env = Environment(
+    loader=FileSystemLoader("src/sendmail"),
+    autoescape=select_autoescape()
+)
+
+def send_email(args):
+    MAILTRAP_API_TOKEN = os.environ.get("MAILTRAP_API_TOKEN")
+    MAILTRAP_SENDER_ADDRESS = os.environ.get("MAILTRAP_SENDER_ADDRESS") 
+    MAILTRAP_TO_ADDRESS = os.environ.get("MAILTRAP_TO_ADDRESS")
+    MAILTRAP_BCC_ADDRESS = os.environ.get("MAILTRAP_BCC_ADDRESS")
```

Comment:
Is it worthwhile to raise an exception early if `MAILTRAP_API_TOKEN`, `MAILTRAP_SENDER_ADDRESS`, or `MAILTRAP_TO_ADDRESS` are unset?

Should there be a `MAILTRAP_CC_ADDRESS` var as well?

Reply:
Good call. That makes sense to add.

Reply:
Added those!